engine/src/main/java/org/terasology/entitySystem/prefab/internal/PrefabFormat.java

```diff
@@ -50,7 +37,10 @@ public PrefabData load(ResourceUrn resourceUrn, List<AssetDataFile> inputs) thro
         try (BufferedReader reader = new BufferedReader(new InputStreamReader(inputs.get(0).openStream(), Charsets.UTF_8))) {
             EntityData.Prefab prefabData = EntityDataJSONFormat.readPrefab(reader);
             if (prefabData != null) {
-                logger.info("Attempting to deserialize prefab {} with inputs {}", resourceUrn, inputs);
+                if (!prefabData.hasName()) {
+                    prefabData = prefabData.toBuilder().setName(resourceUrn.toString()).build();
+                }
+                logger.info("Deserializing prefab {} with inputs {}", resourceUrn, inputs);
```

Comment:
There's a lot of this in the log at start-up. Is it useful as `info` or may we bump it down to `debug`?

Reply:
good point :+1: